docs/tutorials/light-up.md

```diff
@@ -0,0 +1,224 @@
+
+You will also generate a person detector function within your code, to detect a person and trigger the camera.
+
+{{% alert title="Info" color= "info" %}}
+
+Technically, you can detect any object that is listed in the <file>labels.txt</file> (such as a dog or a toilet) but for this tutorial, we are detecting a person.
+
+To detect something else with the camera, just change the string "person" to a different item in the <file>label.txt</file> file.
+
+```python
+               if d.class_name.lower() == "person":
+                   print("This is a person!")
+                   found = True
+```
+
+{{% /alert %}}
+
+The sample code below specifies the file locations of the model and labels, which object you are detecting, and it configures the detector.
+
+```python
+    vision = VisionServiceClient.from_robot(robot)
+    params = {"model_path": "./effdet0.tflite", "label_path": "./labels.txt", "num_threads": 1}
+    personDet = VisModelConfig(name="person_detector", type=VisModelType("tflite_detector"), parameters=params)
+    await vision.add_detector(personDet)
+    names = await vision.get_detector_names()
+    print(names)
+```
+
+Add the host address (for example, 10.1.11.221) of your smart plug that you found in the `kasa discover` step to the code shown below.
+
+In this snippet, if the camera detects a person, it will print to the terminal “This is a person!” and turn on the smart plug.
+If it does not find a person, it will write “There’s nobody here” and will turn off the plug.
+
+```python
+   #example: plug = SmartPlug('10.1.11.221')
+   plug = SmartPlug('replace with the host IP of your plug')
+   await plug.update()
+
+
+   await plug.turn_off()
+   state = "off"
+   for i in range(N):
+       image = await camera.get_image()
+       detections = await vision.get_detections(image, "person_detector")
+       found = False
+       for d in detections:
+           if d.confidence > 0.8:
+               print(d)
+               print()
+               if d.class_name.lower() == "person":
+                   print("This is a person!")
+                   found = True
+       if found:
+           #turn on the smart plug
+           await plug.turn_on()
+           await plug.update()
+           print("turning on")
+           state = "on"
+       else:
+           print("There's nobody here")
+           #turn off the smart plug
+           await plug.turn_off()
+           await plug.update()
+           print("turning off")
+           state = "off"
+```
+
+Next, save and run your code.
+You will see your plug turn on and off as you move in the frame of your computer's webcam!
+
+Your terminal should look like this as your project runs:
+
+```bash
+This is a person!
+turning on
+There's nobody here
+turning off
+```
+
+The complete code for this project can be found [here](https://github.com/viam-labs/devrel-demos/blob/main/Light%20up%20bot/lightupbot.py).

```

Comment:
link to this at the beginning of the blog post as well please

Reply:
It is there in the software section

Reply:
I will add a thing that says create a file and put this in it, and then just change the connect configuration